include/jemalloc/internal/jemalloc_internal_inlines_c.h

```diff
@@ -52,56 +52,81 @@ isalloc(tsdn_t *tsdn, const void *ptr) {
 }
 
 JEMALLOC_ALWAYS_INLINE void *
-iallocztm(tsdn_t *tsdn, size_t size, szind_t ind, bool zero, tcache_t *tcache,
-    bool is_internal, arena_t *arena, bool slow_path) {
+iallocztm_explicit_slab(tsdn_t *tsdn, size_t size, szind_t ind, bool zero,
```

Comment:
For the context, if we are strictly sticking to convention, it's probably worth adding another letter here:
https://github.com/jemalloc/jemalloc/blob/dev/include/jemalloc/internal/jemalloc_internal_inlines_c.h#L37

e.g. `l: accepts a slab parameter`, then this becomes `iallocztml`.

I don't really have a strong preference either way. We talked about switching to a more verbose naming; however being partially-verbose is a bit awkward too. WDYT @Svetlitski ?

Reply:
I personally feel that the current Hungarian notation convention is not the greatest for readability, since all these function names sort of blur together. I'd prefer to keep the longer name, but I'll defer to your judgement.

Reply:
Ok let's keep the long name for now. Next time when we are changing any of the params, it'd be the signal to commit to either the long or short format.